css/port.css

```diff
@@ -236,142 +336,325 @@ section .aim ul li {
   margin: 10px 0;
 }
 
-.btn-resume {
-  text-decoration: none;
+.pot1 {
+  top: 780px;
+  margin-top: 130px;
+}
+
+.pot2 {
+  top: 780px;
+  margin-top: 130px;
+}
+
+.pot3 {
+  top: 780px;
+  margin-top: 130px;
+}
+
+/* ---Portfolio----- */
+@media (max-width: 470px) {
```

Comment:
- [ ] Kindly remove this media query for the mobile version. Make sure to use media queries for only Desktop breakpoint in the CSS file.